docs/resources/hss_host_protection.md

```diff
@@ -51,6 +51,9 @@ The following arguments are supported:
   If omitted, randomly select quota for the corresponding version.
   This field is valid only when `charging_mode` is set to **prePaid**.
 
+* `is_wait_host_available` - (Optional, Bool) Specifies whether to wait for the host agent status to become **online**.
+  The value can be **true** or **false**.
```

Comment:
Defaults to **false**.

-> If this field is set to **true**, the program will wait for `30` minutes until the status of `host_id` changes to online before switching the protection status.

Reply:
done.